.eslintrc.json

```diff
@@ -4,7 +4,8 @@
     "browser": true,
     "commonjs": true,
     "node": true,
-    "es2021": true
+    "es2021": true,
+    "jest": true
```

Comment:
So the `describe`, `it`, `expect` are defined globally by `jest`. Here we add `jest` to the `env`, so ESLint won't throw error due to `no-undef` rule: https://eslint.org/docs/rules/no-undef